x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/node_square.tsx

```diff
@@ -112,51 +102,96 @@ export const NodeSquare = ({
   color: string;
   nodeName: string;
   value: string;
-  nodeBorder?: CSSProperties;
+  showBorder?: boolean;
 }) => {
   const valueMode = squareSize > 70;
   const ellipsisMode = squareSize > 30;
   const nodeAriaLabel = i18n.translate('xpack.infra.node.ariaLabel', {
     defaultMessage: '{nodeName}, click to open menu',
     values: { nodeName },
   });
+  const style: CSSProperties | undefined = showBorder ? { border: 'solid 4px #000' } : undefined;
 
   return valueMode || ellipsisMode ? (
-    <NodeContainer
+    <div
+      css={css`
+        position: relative;
+        cursor: pointer;
+      `}
       data-test-subj="nodeContainer"
       style={{ width: squareSize || 0, height: squareSize || 0 }}
       onClick={togglePopover}
+      onKeyPress={togglePopover}
+      onFocus={showToolTip}
```

Comment:
Why do we need these?

Reply:
It was an accessibility eslint error: onMouseOver must be accompanied by onFocus for accessibility.eslint [jsx-a11y/mouse-events-have-key-events](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/mouse-events-have-key-events.md)